assignment-day-2/src/sql/queries/extract_employee_data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO raw_employee(employee_id, first_name, last_name, department_id, 
+department_name, manager_employee_id, employee_role, salary, hire_date, 
+terminated_date, terminated_reason, dob ,fte ,location) 
+VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s);
```

Comment:
```suggestion
VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s);

```
Please add a new line after the end in each file